frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -74,7 +86,28 @@ const mapDispatchToProps = {
 
 @connect(mapStateToProps, mapDispatchToProps)
 @fitViewport
-@title(({ dashboard }) => dashboard && dashboard.name)
+@title(
+  ({
+    dashboard,
+    isLoadingDashCards,
+    totalDashCards,
+    cardsLoaded,
+    hasBeenSeen,
+    isLoadingDashCardsComplete,
+  }) => {
+    if (isLoadingDashCards) {
+      return {
+        title: `${cardsLoaded} / ${totalDashCards} loaded`,
+        titleIndex: 1,
+      };
+    }
+    if (isLoadingDashCardsComplete && !hasBeenSeen) {
+      return "Your Dashboard is Ready";
```

Comment:
Please use the `t` function from `ttag` to translate strings. This should look like:

```js
import { t } from "ttag";

return t`Your Dashboard is Ready`
```